README.md

```diff
@@ -176,13 +177,12 @@ for suggestions on what could be wrong and how to fix it.
 
 ## Changing the file system
 
-In Mbed OS, a C++ classes that inherits from the [FileSystem](https://os.mbed.com/docs/latest/reference/storage.html#declaring-a-file-system)
-interface represents each file system. You can change the file system in the
-example by changing the class declared in main.cpp.
+In Mbed OS, C++ classes that inherit from the [FileSystem](https://os.mbed.com/docs/latest/reference/storage.html#declaring-a-file-system)
```

Comment:
Check the links - they shoudl reference the API in each case:
https://os.mbed.com/docs/mbed-os/latest/apis/filesystem.html
https://os.mbed.com/docs/mbed-os/latest/apis/fatfilesystem.html